charts/flux2/templates/pre-install-job.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}-pre"
```

Comment:
```suggestion
  name: "{{ .Release.Name }}-flux-check"
```

Reply:
Thanks for the comment, they are resolved.